targets/android-studio

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+REQUIRES='xiwi gtk-extra'
+DESCRIPTION='Installs Android Studio as an easily launchable xiwi application'
+HOSTBIN='startandroidstudio'
+
+### Append to prepare.sh:
+mkdir /etc/bash-completion.d #Must do this in order to avoid package management error while installing umake
+if [ $DISTRO != "ubuntu" ]; then
```

Comment:
If you're going the PPA approach for non-ubuntu, you should do something more like [this](https://github.com/dnschneid/crouton/blob/90ad47ff22e148db41ba627992ee5cc5d0c14175/targets/xfce#L13).